aiida/backends/tests/export_and_import.py

```diff
@@ -1481,6 +1483,175 @@ def get_all_node_links(self):
                   edge_project=['label', 'type'], output_of='input')
         return qb.all()
 
+    class ComplexGraph:
```

Comment:
this file already has >1k lines
could you please move the ComplexGraph out into a separate file?

Reply:
I propose this to be done under another PR. And mainly to 1.0
There is already one created by Casper (#2702)